src/Domains/Social/Interactions/Models/UsersInteractions.php

```diff
@@ -45,4 +45,9 @@ public function getCacheKey(): string
     {
         return Str::simpleSlug($this->entity_namespace) . '-' . $this->entity_id;
     }
+
+    public function entity(): BelongsTo
```

Comment:
@rwhite27 remember you dont need this relationship, no? we already have it on the other trait